docs/pages/guides/authentication/express.mdx

```diff
@@ -70,6 +88,22 @@ app.post("/api/auth", (req, res) => {
       res.status(403).end();
     });
 });
+
+```
+The `userId` used in the `authorize` function corresponds to userId used in our APIs when setting permissions (e.g. in create room).
+You can use the following example to use data from `userId` and `userInfo on the front end.
```

Comment:
Seems like you're missing a closing quote here. Looks like this is also copy/paste to other files:
```suggestion
You can use the following example to use data from `userId` and `userInfo` on the front end.
```